### ISSUE TITLE
Clarify software and cloud service responsibilities with Stena

### ISSUE THREAD

**Issue by Vivesoll** [Power2U/ICT_Task_Force#58]

**Comment by Vivesoll** [comment]
➤ Albin Engholm commented:

Here I think we all will need to help out [Johan Norberg](https://app.asana.com/0/264682084233526/264682084233526)[Maksym](https://app.asana.com/0/221679536481849/221679536481849)[Kristofer Sjöholm](https://app.asana.com/0/253020614776456/253020614776456)[Dapeng Lan](https://app.asana.com/0/226761305033663/226761305033663).

One idea could be to use Web Port's user agreement as a basis. 

Let's start on this task once the consultancy contract is in place and we have more information on the exact requirements from the pilots

**Comment by Vivesoll** [comment]
➤ 233481402429721 commented:

Follow up with a plan after Stena start up meeting

**Comment by Vivesoll** [comment]
➤ Albin Engholm commented:

Make initial proposal and discuss with Stena (maybe not needed for the pilot)

**Comment by Vivesoll** [comment]
➤ Albin Engholm commented:

The crucial thing for the pilot is more on repsonsibility, errors, maintenance, code maintenance, etc. 
also how they can use the code base for the future

**Comment by Vivesoll** [comment]
➤ Albin Engholm commented:

Albin and Kristofer decides on a strategy for this

**Comment by Vivesoll** [comment]
➤ Albin Engholm commented:

Kristofer checks for exising förvaltningsavtal form Sweco

**Comment by Vivesoll** [comment]
➤ Kristofer Sjöholm commented:

I have three different agreements that Sweco and SVT made that I can share to [Albin Engholm]([Albin Engholm](https://app.asana.com/0/193610864030898/193610864030898)), but will not do it here because they contain parts that I don't have rights to share.

**Comment by Vivesoll** [comment]
➤ Kristofer Sjöholm commented:

Meeting next friday 27th about this?
